docs/source/conditions.rst

```diff
@@ -252,6 +252,15 @@ Here's the list of pull request attribute that can be used in conditions:
    * - ``title``
      - string
      - The title of the pull request.
+   * - ``time``
+     - string
+     - A time in format ``HH:MM``.
```

Comment:
This does not make any sense. What time it is?
Time of creation? Time of update?

Reading the code I think you want this to be named `now`, no?

Reply:
If we go with `now`, then it should be `now-time`, to also have `now-day`, `now-month`, `now-day-of-week`.

Reply:
Ah. We're back at "naming things is hard". 😆 

what about `current-time` and `current-day`, `current-month`… ? I'm stealing this naming from PostgreSQL.